Userland/Applications/MailSettings/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+serenity_component(
+        MailSettings
+        REQUIRED
+        TARGETS MailSettings
+)
+
+compile_gml(MailSettingsWindow.gml MailSettingsWindowGML.h mail_settings_window_gml)
+
+set(SOURCES
+        main.cpp
+        MailSettingsWindow.cpp
+        MailSettingsWindow.h
+        MailSettingsWindowGML.h
+        )
```

Comment:
This should have four spaces for indentation and the closing `)` should be on the same level like the `set(`.